recipe/meta.yaml

```diff
@@ -7,30 +7,28 @@ package:
 source:
     fn: Simbody-{{ version }}.tar.gz
     url: https://github.com/simbody/simbody/archive/Simbody-{{ version }}.tar.gz
-    sha256: 7716d6ea20b950e71e8535faa4353ac89716c03fd7a445dd802eb6a630796639
+    sha256: d371a92d440991400cb8e8e2473277a75307abb916e5aabc14194bea841b804a
 
 build:
-    number: 1003
+    number: 0
     run_exports:
         - {{ pin_subpackage('simbody', max_pin='x.x') }}
-    # NOTE : For C++11 builds, vc14/VS 2015 are the only acceptable targets
-    # which only Python 3.5 supports. So skip previous Python versions.
-    skip: true                # [win and vc<14]
     # NOTE : This is needed to ensure the paths to Simbody's binaries, like
     # simbody-visualizer, are corrected.
     detect_binary_files_with_prefix: true
 
 # NOTE: This installs Simbody and the optional visualizer on Linux, Windows, and
 # OSX. We build against the openblas available in conda-forge which includes
-# lapack on Linux and OSX. blas and lapack are not needed on Windows because
-# Simbody includes them. For the visualizer, FreeGLUT is available via
+# lapack on Linux and OSX. For the visualizer, FreeGLUT is available via
 # anaconda.org. Glut is included with Simbody for Windows and OSX includes a
 # Glut implementation.
 # https://github.com/AnacondaRecipes/r-rgl-feedstock/blob/master/recipe/meta.yaml#L48-L67
```

Comment:
Not sure this is all relevant anymore as I think conda forge packages freeglut now for all OSs: https://github.com/conda-forge/freeglut-feedstock